Add PULL_REQUEST_TEMPLATE.md

denyeart opened the pull request:
Add pull request template for contributors to complete.
Includes a checklist of common considerations.

Signed-off-by: David Enyeart <enyeart@us.ibm.com>

lehors commented:
LGTM. The only thing that I might add is something about the title of the PR. For the labs we've seen people submit PRs with very unhelpful titles. So, we added to the PR template that one should make the title descriptive. Might seem obvious but it evidently wasn't to everyone...

denyeart commented:
> LGTM. The only thing that I might add is something about the title of the PR. For the labs we've seen people submit PRs with very unhelpful titles. So, we added to the PR template that one should make the title descriptive. Might seem obvious but it evidently wasn't to everyone...

@lehors Thanks, I've updated the first comment to be:
"Provide a descriptive summary of your changes in the Title above"

btl5037 commented:
/azp help
